modules/Kconfig.mcuboot

```diff
@@ -0,0 +1,6 @@
+config SIGN_IMAGES
```

Comment:
That option will only be available for the MCUBoot image. This option (`SIGN_IMAGES`) is used for the parent image, which might not have access to MCUBoot configurations (for instance if its not built from source)

Reply:
> MCUboot already contains BOOT_SIGNATURE_TYPE_NONE option. which is for disabling image signature check. MCUboot check just image hash.

We don't want to disable signature check, we want to disabling signing of the images as part of the build system.
So that users can built the images and then sign them later.

See also: NCSDK-7277

Reply:
understor